pymc/logprob/transforms.py

```diff
@@ -691,9 +710,8 @@ def find_measurable_transforms(fgraph: FunctionGraph, node: Node) -> Optional[Li
         if measurable_input_idx != 0:
             return None
         try:
-            (power,) = other_inputs
-            power = pt.get_underlying_scalar_constant_value(power).item()
-        # Power needs to be a constant
+            power = pt.get_underlying_scalar_constant_value(node.inputs[1]).item()
```

Comment:
I find it more readable in the old version, where we see we take it from the other inputs